Makefile

```diff
@@ -6,26 +6,54 @@
 # This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not
 # distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
 #
-# date: 2015-10-08
+# date: 2015-10-11
 #
 
 #-----------------------------------------------------------------------------------------------------------------------
+# make control/macros
+#-----------------------------------------------------------------------------------------------------------------------
+# defaults
+mk_do_inc := 1
```

Comment:
Let's stay with FULL_CAPS and full words here, with no prefixes. I'm probably going to change the variables you mentioned - the code in PARSE_SUBDIRECTORY is based on some other Makefiles I've seen, and the variables in this particular macro are very very special, because they keep changing their value, which is different from other variables in the make build system of distortos.